roles/jd-server/src/main.rs

```diff
@@ -111,11 +111,10 @@ async fn main() {
     let mempool_cloned_ = mempool.clone();
     let (status_tx, status_rx) = unbounded();
     let sender = status::Sender::Downstream(status_tx.clone());
-    let empty_mempool_log_counter = 10 / mempool_update_interval.as_secs();
+    let mut last_empty_mempool_warning = std::time::Instant::now();
```

Comment:
I think you want to set this in the past, otherwise it won't warn the first time.